<launch>
<node name="unity_velocity_control_wrapper" pkg="px4_velocity_control_wrapper" type="unity_velocity_control_wrapper" output="screen">
      <param name="px4_ref_vel_msg_pub_topic" value="/mavros/setpoint_velocity/cmd_vel_unstamped"/>        <!--The Target Velocity Message to PX4-->
      <param name="tgt_vel_msg_sub_topic" value="/other_controller/target_velocity"/>                      <!--The Target Velocity from Other Controller-->
      <param name="cmd_reset_time" value="1.0"/> <!--1.0s-->
      <param name="run_frequen_hz" value="20.0"/>
      <param name="network_data_case_pub_topic" value="/network/data/get"/>
  </node>
</launch>
